eventhubs/data-plane/pom.xml

```diff
@@ -1,84 +1,79 @@
 <!-- Copyright (c) Microsoft Corporation. All rights reserved.
      Licensed under the MIT License. -->
 <project xmlns="http://maven.apache.org/POM/4.0.0"
-		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
-							 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-	<modelVersion>4.0.0</modelVersion>
-	<groupId>com.microsoft.azure</groupId>
-	<artifactId>azure-eventhubs-clients</artifactId>
-	<packaging>pom</packaging>
-	<version>2.0.0</version>
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+                             http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.microsoft.azure</groupId>
+  <artifactId>azure-eventhubs-clients</artifactId>
+  <packaging>pom</packaging>
+  <version>2.0.0</version>
 
-	<name>Microsoft Azure Event Hubs SDK Parent</name>
-	<description>Java libraries for talking to Windows Azure Event Hubs</description>
-	<url>https://github.com/Azure/azure-event-hubs</url>
+  <name>Microsoft Azure Event Hubs SDK Parent</name>
+  <description>Java libraries for talking to Windows Azure Event Hubs</description>
+  <url>https://github.com/Azure/azure-sdk-for-java</url>
 
-	<parent>
-		<groupId>com.azure</groupId>
-		<artifactId>azure-client-sdk-parent</artifactId>
-		<version>1.0.0-SNAPSHOT</version>
-		<relativePath>../../pom.client.xml</relativePath>
-	</parent>
+  <parent>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-client-sdk-parent</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <relativePath>../../pom.client.xml</relativePath>
+  </parent>
 
-	<distributionManagement>
-		<site>
-			<id>azure-java-build-docs</id>
-			<url>${site.url}/site/${project.artifactId}</url>
-		</site>
-	</distributionManagement>
+  <distributionManagement>
+    <site>
+      <id>azure-java-build-docs</id>
+      <url>${site.url}/site/${project.artifactId}</url>
+    </site>
+  </distributionManagement>
 
-	<scm><url>https://github.com/Azure/azure-sdk-for-java</url></scm>
+  <scm>
+    <url>scm:git:https://github.com/Azure/azure-sdk-for-java</url>
+  </scm>
 
-	<properties>
-		<proton-j-version>0.31.0</proton-j-version>
-		<qpid-proton-j-extensions-version>1.1.0</qpid-proton-j-extensions-version>
-	</properties>
+  <properties>
+    <proton-j-version>0.31.0</proton-j-version>
+    <qpid-proton-j-extensions-version>1.1.0</qpid-proton-j-extensions-version>
+  </properties>
 
-	<repositories>
-		<repository>
-			<id>HTTP proxy server in java</id>
-			<url>http://raw.github.com/SreeramGarlapati/jproxy/master/releases</url>
-		</repository>
-	</repositories>
+  <repositories>
+    <repository>
+      <id>HTTP proxy server in java</id>
+      <url>http://raw.github.com/SreeramGarlapati/jproxy/master/releases</url>
+    </repository>
+  </repositories>
```

Comment:
Please discuss this with Sreeram to understand better.

Reply:
Sreeram's comment:

"This was added for proxy related unit tests. This jar helps hosts an in-memory http proxy.
If this reference is not working – then, it might no longer be used. You may find this out by running the tests…
If this is needed and needs to be fixed, then, this is where the original jar is: https://github.com/SreeramGarlapati/jproxy/tree/master/releases/org/jutils/jproxy/jproxy/0.0.1 
"

Updated the jproxy link .

Reply:
Can it be placed somewhere more convenient - e.g. released in Maven Central?

Reply:
@mssfang : Should this be in a more central location? Did Sreeram consider that?